app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -339,6 +339,7 @@ function AcuantCapture(
   const { isMobile } = useContext(DeviceContext);
   const { t, formatHTML } = useI18n();
   const [captureAttempts, incrementCaptureAttempts] = useCounter(1);
+  const selfieAttempts = useRef(1);
```

Comment:
What does the `1` do here?

I wonder if this is adding the extra attempt.

Reply:
@night-jellyfish , tried to be consistent with `captureAttempts` which has a default 1. Was using `0` here, if this is the consent I can use `0`.

Reply:
I see. I would be curious what other folks think / see.

Personally I think a default of `1` is fine, but not if it increases the actual number of attempts to one extra than it actually is.

Reply:
@night-jellyfish , set to 0 now.